views/openacademy_course_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="course_view_form" model="ir.ui.view">
+        <field name="name">openacademy.course.view.form</field>
+        <field name="model">openacademy.course</field>
+        <field name="arch" type="xml">
+            <form string="Course">
+                <sheet>
+                    <group>
+                        <field name="name" />
+                        <field name="responsible_id" />
+                    </group>
+                </sheet>
+                <notebook>
+                    <page name="description" string="Description">
+                        <field name="description" nolabel="1"/>
+                    </page>
+                </notebook>
+            </form>
+        </field>
+    </record>
+
+    <record id="course_view_search" model="ir.ui.view">
+        <field name="name">openacademy.course.view.search</field>
+        <field name="model">openacademy.course</field>
+        <field name="arch" type="xml">
+            <search string="Courses">
+                <field name="name"/>
+                <field name="description"/>
+            </search>
+        </field>
+    </record>
+
+    <record id="course_view_tree" model="ir.ui.view">
+        <field name="name">openacademy.course.view.tree</field>
+        <field name="model">openacademy.course</field>
+        <field name="arch" type="xml">
+            <tree string="Courses">
```

Comment:
string is deprecated in tree views, thus ignored:

```suggestion
            <tree>
```

Same for the other tree view.

Quoting [the guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#23views):

> For v8 and above it is recommended to avoid using the string attribute on list views (<tree>) which has been deprecated and is no longer displayed.

Reply:
I'm going to fix it.